README.md

```diff
@@ -137,6 +137,24 @@ To use the Docker image, run:
 
 	docker run --rm -v $PWD:/mnt -w /mnt my:tag <shfmt arguments>
 
+### pre-commit
+
+It is possible to use shfmt with [pre-commit][pre-commit] and a `local`
+repo configuration like:
+
+```yaml
+  - repo: local
+    hooks:
+      - id: shfmt
+        name: shfmt
```

Comment:
do we need a name? it adds nothing beyond the id

Reply:
Yes, it's mandatory. https://pre-commit.com/#new-hooks

Reply:
This can be added to the bucket list of an "unnecessary verbosity" bug report :)